charts/amoro/tests/amoro-database-secret_test.yaml

```diff
@@ -0,0 +1,86 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
+#    or more contributor license agreements.  See the NOTICE file
+#    distributed with this work for additional information
+#    regarding copyright ownership.  The ASF licenses this file
+#    to you under the Apache License, Version 2.0 (the
+#    "License"); you may not use this file except in compliance
+#    with the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+
+suite: Test Amoro Database Secret
+templates:
+  - amoro-database-secret.yaml
+chart:
+  appVersion: 0.1.0
+  version: 0.6.0
+tests:
+  - it: Amoro database secret should not exist when database type set to derby
+    set:
+      amoroConf:
+        database:
+          type: derby
+    asserts:
+      - hasDocuments:
+          count: 0
+  - it: Amoro database secret should exist when database type set to mysql
```

Comment:
when database type is set to mysql

Reply:
Some similar contents in this file can be changed

Reply:
fixed.